packages/smartgpt-bridge/src/logging/index.js

```diff
@@ -138,7 +138,6 @@ var __generator =
   };
 Object.defineProperty(exports, "__esModule", { value: true });
 exports.mongoChromaLogger = mongoChromaLogger;
```

Comment:
js module in src folder, I probably missed these while I was initially moving the code over.